docs/source/intro/installation.rst

```diff
@@ -391,19 +394,24 @@ To leave or deactivate the environment, simply run:
 .. admonition:: Update install tools
    :class: tip title-icon-tip
 
-   You may need to install ``pip`` and ``setuptools`` in your virtual environment in case the system or user version of these tools is old
+   You may need to install ``pip`` and ``setuptools`` in your virtual environment, in case the system or user version of these tools is old.
 
    .. code-block:: console
 
       (aiida) $ pip install -U setuptools pip
 
-If you have `Conda`_ installed then you can directly create a new environment with ``aiida-core`` and (optionally) Postgres and RabbitMQ installed.
+Conda
+-----
+
+If you have `Conda`_ installed then you can directly create a new environment with ``aiida-core`` and (optionally) Postgres and RabbitMQ services installed.
 
 .. code-block:: console
 
    $ conda create -n aiida -c conda-forge python=3.7 aiida-core aiida-core.services pip
-   $ conda activate
-   $ conda deactivate aiida
+   # to activate
```

Comment:
Are these comments really necessary? If people cannot decipher what `conda activate` and `conda deactivate` do, I think they are a lost cause :sweat_smile:

Reply:
I don't know, after tutorials and countless github issues interactions, I'm begin to lose faith on what is "obvious" or not 😆 
(but yeh I've removed them)